plugins/logs/plugin.go

```diff
@@ -227,13 +229,13 @@ type ReportingConfig struct {
 
 // Config represents the plugin configuration.
 type Config struct {
-	Plugin        *string         `json:"plugin"`
-	Service       string          `json:"service"`
-	PartitionName string          `json:"partition_name,omitempty"`
-	Reporting     ReportingConfig `json:"reporting"`
-	MaskDecision  *string         `json:"mask_decision"`
-	ConsoleLogs   bool            `json:"console"`
-
+	Plugin          *string         `json:"plugin"`
+	Service         string          `json:"service"`
+	PartitionName   string          `json:"partition_name,omitempty"`
+	Reporting       ReportingConfig `json:"reporting"`
+	MaskDecision    *string         `json:"mask_decision"`
+	ConsoleLogs     bool            `json:"console"`
+	Resource        string          `json:"resource"`
```

Comment:
Nit: make this a `*string` so we can tell the difference between an empty string and unset. Check if `c.Resource` is nil below in the defaulting logic.

Reply:
fixed!